config/application.rb

```diff
@@ -23,10 +23,41 @@
 
 module PracticalDeveloper
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1 # NOTE: [Rails 6] we should at least work towards updating this to 5.2
+    # Specify the default Rails settings version we're targetting
+    # See: https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults
+    config.load_defaults 6.0
+
+    ### FRAMEWORK DEFAULT OVERRIDES
+    # Override new framework defaults to keep existing behavior.
+    #
+    # NOTE: For booleans the new default is the opposite of what we're setting here. For other
+    # options, the new default is mentioned in a comment. Once we're ready to enable a new default
+    # setting we can remove the line here.
+
+    ## Rails 5.0
+    # There is no easy way to use per-form tokens and view caching at the same time.
+    # Therefore we disable "per_form_csrf_tokens" for the time being.
+    config.action_controller.per_form_csrf_tokens = false
+
+    ## Rails 6.0
+    # Determines whether forms are generated with a hidden tag that forces older versions of Internet
+    # Explorer to submit forms encoded in UTF-8
+    config.action_view.default_enforce_utf8 = true
+
+    # Enables the same cache key to be reused when the object being cached of type ActiveRecord::Relation
+    # changes by moving the volatile information (max updated at and count) of the relation's cache
+    # key into the cache version to support recycling cache key.
+    config.active_record.collection_cache_versioning = false
```

Comment:
Is this just the default value? Is there a reason we can't enable this?

Or am I misreading?

Reply:
1. If it's in this file it's because we're overriding the default, see the comment further up in the same file.
2. This PR does not attempt to change any current behavior. It's only about changing the Rails version we pass to `load_defaults` and consolidating the config files.
3. If you don't mind, I'd like to change the actual setting in a follow-up PR, it should be safe to do but I want to make it easier for us to debug things in case anything goes wrong, which is why I'd like to deploy the cleanup PR that doesn't actually change any settings first.